plone/app/layout/sitemap/sitemap-index.xml

```diff
@@ -0,0 +1,11 @@
+<?xml version="1.0" encoding="utf-8"?>
+<sitemapindex xmlns="http://www.sitemaps.org/schemas/sitemap/0.9"
+              xmlns:tal="http://xml.zope.org/namespaces/tal"
+              xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+              xsi:schemaLocation="http://www.sitemaps.org/schemas/sitemap/0.9 http://www.sitemaps.org/schemas/sitemap/0.9/sitemap.xsd"
+>
+  <sitemap tal:repeat="page view/sitemap_count">
+    <loc tal:content="string:${view/navroot_url}/sitemap.xml.gz?page=${page}">
```

Comment:
I think we can use context/@@plone_portal_state/navigation_root_url instead of view/navroot_url. Less code, we could remove the `navroot_url` function in `sitemap.py`.